skore/src/skore/_sklearn/_plot/data/table_report.py

```diff
@@ -0,0 +1,697 @@
+                [True, False],
+                [
+                    pd.Series(contingency_table.columns),
+                    pd.Series(contingency_table.index),
+                ],
+                strict=False,
+            ):
+                _resize_categorical_axis(
+                    figure=self.figure_,
+                    ax=self.ax_,
+                    n_categories=sbd.n_unique(x_or_y),
+                    is_x_axis=is_x_axis,
+                    size_per_category=size_per_category,
+                )
+
+        sns.despine(self.figure_, **despine_params)
+
+        self.ax_.set(xlabel=sbd.name(x), ylabel=sbd.name(y))
+        if self.ax_.legend_ is not None:
+            sns.move_legend(self.ax_, (1.05, 0.0))
+
+    def _plot_cramer(self, *, heatmap_kwargs: dict[str, Any] | None) -> None:
+        """Plot Cramer's V correlation among all columns.
+
+        Parameters
+        ----------
+        heatmap_kwargs : dict, default=None
+            Keyword arguments to be passed to seaborn's heatmap.
+        """
+        if heatmap_kwargs is None:
+            heatmap_kwargs = self._default_heatmap_kwargs or {}
+
+        heatmap_kwargs_validated = _validate_style_kwargs(
+            {
+                "xticklabels": True,
+                "yticklabels": True,
+                "robust": True,
+                "square": True,
+                "annot": True,
+                "annot_kws": {"size": 10},
+                "fmt": ".2f",
+            },
+            heatmap_kwargs or {},
+        )
+
+        cramer_v_table = pd.DataFrame(
+            _column_associations._cramer_v_matrix(self.summary["dataframe"]),
+            columns=self.summary["dataframe"].columns,
+            index=self.summary["dataframe"].columns,
+        )
+        # only show the lower triangle of the heatmap since it is a correlation matrix
+        # and keep the diagonal as well.
+        mask = np.triu(np.ones_like(cramer_v_table, dtype=bool), k=1)
+
+        sns.heatmap(cramer_v_table, mask=mask, ax=self.ax_, **heatmap_kwargs_validated)
+        self.ax_.set(title="Cramer's V Correlation")
+
+    def frame(
+        self, *, kind: Literal["dataset", "top-associations"] = "dataset"
+    ) -> pd.DataFrame:
+        """Get the data related to the table report.
+
+        Parameters
+        ----------
+        kind : {'dataset', 'top-associations'}
+            The kind of data to return.
+
+        Returns
+        -------
+        DataFrame
+            The dataset used to create the table report.
+        """
+        if kind == "dataset":
+            return self.summary["dataframe"]
+        elif kind == "top-associations":
+            return pd.DataFrame(self.summary["top_associations"])
+        else:
+            raise ValueError(f"Invalid kind: {kind!r}")
+
+    def _html_repr(self) -> str:
+        """Show the HTML representation of the report."""
+        return to_html(self.summary, standalone=False, column_filters=None)
```

Comment:
I think that it remains only this line to be covered. We could do similarly to https://github.com/skrub-data/skrub/blob/main/skrub/_reporting/tests/test_table_report.py#L52-L64 to check the output of this function.